### PR TITLE
Feature/enable multiple isins

### PR DESCRIPTION
**Pull request by simaob**
This small PR allows the input of multiple isin codes for the same company.

- render as strings;
- using a tags input to manage them;
- updating the importer to allow saving multiple isins, matching the import on the company name.

### CODE REVIEW

File: app/services/import/companies.rb [hunk 1]
```diff
@@ -17,7 +17,14 @@ def call
 
     def import
       import_each_with_logging(csv, FILEPATH) do |row|
-        company = Company.find_or_initialize_by(isin: row[:isin])
+        company = Company.find_or_initialize_by(name: row[:company_name])
+
+        company.isin = if company.isin.present?
```
Comment (kowal) [inline app/services/import/companies.rb]: couldn't this be moved to `company_attributes` method?

Comment (simaob) [reply app/services/import/companies.rb]: probably so.  I kept it here, but it doesn't make so much sense now. I'll move it.

File: app/models/company.rb [hunk 2]
```diff
@@ -48,4 +48,8 @@ def latest_assessment
   def to_s
     name
   end
+
+  def isin_string
```
Comment (kowal) [inline app/models/company.rb]: I think this belongs to `CompanyDecorator` which needs to be created :)

Comment (kowal) [reply app/models/company.rb]: also, this name is a bit confusing - this returns an array.

File: app/services/import/companies.rb (hunk 1 above)
Comment (simaob) [reply app/services/import/companies.rb]: hmmm, actually, I would need to pass the current isin to that method if I were to manage this there. Currently the `company_attributes` is not aware of the record being updated. So not sure if that's worth it.

File: app/models/company.rb (hunk 2 above)
Comment (simaob) [reply app/models/company.rb]: you're right! I've moved the method to a new decorator, and I've renamed it to `isin_as_tags`. 🙇

Comment (kowal) [reply app/models/company.rb]: 👍

**Pull request merged**